apps/node/server.js

```diff
@@ -0,0 +1,138 @@
+const express = require("express");
+const axios = require("axios");
+const dotenv = require("dotenv").config();
+const crypto = require("crypto");
+const qs = require("qs");
+
+const ATTENTIVE_API_URL = "https://api.attentivemobile.com/v1";
+const ACCESS_TOKEN_ENDPOINT =
+  "https://api.attentivemobile.com/v1/authorization-codes/tokens";
+
+const CLIENT_ID = dotenv.parsed.CLIENT_ID; // Your client id
+const CLIENT_SECRET = dotenv.parsed.CLIENT_SECRET; // Your secret
+const REDIRECT_URI = dotenv.parsed.REDIRECT_URI; // Your redirect uri
+const PORT = 3002;
+
+let data = {};
```

Comment:
Any other name suggestions? datastore/database/temp are all really awkward.

Reply:
I'd suggest moving it to its own "dal" module, and exporting "getState"/"setState" from that